### DISCUSSION TITLE
Change colors of argon cursor or slider ball

### DISCUSSION THREAD

**Discussion by Kaerulans** [ppy/osu#21880] [Ideas] [answered]
I constantly lose track of my cursor during sliders when using argon because both the slider ball and the cursor are red. I feel like one of them should have its color changed to avoid this.

![Screenshot from 2022-12-28 13-25-11](https://user-images.githubusercontent.com/55179197/209797771-2c10fc8d-5365-44e0-a3f1-07b6530095c8.png)

**Comment by frenzibyte** [comment, marked as answer]
#21986
